src/styles/blocks/stars/separate-star.scss

```diff
@@ -0,0 +1,19 @@
+@for $i from 0 through 5 {
+  @if #{$i} == 0 {
+    .stars--#{$i} :nth-child(1n + 1) {
+      background-image: url('../images/star.svg');
+    }
+  } @else if #{$i} == 5 {
```

Comment:
This logic is overcomplicated. Use index to set the active stars. 
Also, it should be `stars` block

Reply:
I don't understand about the index. 
I was trying to use only properties which are necerally to use.

Reply:
Simplify this part. And move it to `stars` CSS block
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/96881fc1-041f-40f2-a388-f0f63fd1d46e)

Reply:
@polosanya you want me to color all stars grey color and then color only the necessary ones?